sdks/go/pkg/beam/core/graph/coder/coder.go

```diff
@@ -246,7 +246,7 @@ func NewBytes() *Coder {
 
 // NewVarInt returns a new int32 coder using the built-in scheme.
```

Comment:
Looks like you missed changing this to say "int64 coder"

Reply:
Good catch! Updated in my client. I have a small set of other cleanup changes that it can go in with.